docs-tool/src/components/Page.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Prism from 'prismjs';
+
+import { getNextStory, getPreviousStory } from 'glutenberg';
+import markdown from '../markdown';
+
+class Tabs extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			activeTab: 0,
+		};
+	}
+
+	selectTab( index ) {
+		return () => {
+			this.setState( { activeTab: index } );
+		};
+	}
+
+	componentDidUpdate() {
+		Prism.highlightAll();
```

Comment:
Can we be more targeted about this? Either passing the individual node or the code content?